test/simulator/abilities/battlearmor.js

```diff
@@ -11,35 +11,32 @@ describe('Battle Armor', function () {
 	});
 
 	it('should prevent moves from dealing critical hits', function () {
-		battle = common.createBattle([
-			[{species: 'Slowbro', ability: 'battlearmor', moves: ['quickattack']}],
-			[{species: 'Cryogonal', ability: 'noguard', moves: ['frostbreath']}],
-		]);
+		battle = common.createBattle();
+		battle.join('p1', 'Guest 1', 1, [{species: 'Slowbro', ability: 'battlearmor', moves: ['quickattack']}]);
+		battle.join('p2', 'Guest 2', 1, [{species: 'Cryogonal', ability: 'noguard', moves: ['frostbreath']}]);
```

Comment:
Uh, passing them in as an array is preferred. Although you should probably see my previous test work for preferred indentation.

Reply:
https://github.com/Zarel/Pokemon-Showdown/commit/ead4ffff642fa6f75f2d10aa1f5002c84405b176#diff-089f01cef07cb19599fb863a831f3255R102

Reply:
Do you want me to make _every_ test do that, too, or just to leave these ones as they were?

Reply:
If you're going to change them, you might as well change them to the new standard; that's all I'm sa ying.

Reply:
poke?